### PR TITLE
Make qemuarm32 work

### PR DESCRIPTION
**Pull request by vasilvas99**
PR to integrate the qemuarm32 target and make it build/run/switch rauc slots correctly.

Most of the changes are direct transfers from the Qemuarm64 target

### CODE REVIEW

**Comment by vasilvas99** [comment]
# Notes

- Qemuarm (32bit) now a fully build-able target 
- A new leda-qemuarm.yaml has to be created in leda distro
- Since there is no SSTATE cache for QEMUARM (32bit) the build has to be done without mirrors, otherwise it fails.

File: meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Eclipse Leda - fstab for qemuarm RAUC redundant boot setup
+# Eclipse Leda - fstab for qemuarm64 RAUC redundant boot setup
```
Comment (stlachev) [inline meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab:2]: qemuarm64 -> qemuarm

Comment (vasilvas99) [reply meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab:2]: Fixed in latest version in file: https://github.com/SoftwareDefinedVehicle/meta-leda-fork/blob/make-qemuarm32-work/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab#L1

**Review by stlachev — APPROVED** [review]
LGTM

**Pull request merged**